Sources/ArcGISToolkit/Components/Offline/OfflineManager.swift

```diff
@@ -140,60 +137,32 @@ public class OfflineManager: ObservableObject {
         defer { self.offlineMapInfos = infos }
         guard let contents = try? FileManager.default.contentsOfDirectory(atPath: url.path()) else { return }
         for dir in contents {
-            guard let info = makeOfflineMapInfo(url: url.appending(path: dir)) else { continue }
+            guard let info = OfflineMapInfo.make(from: url.appending(path: dir)) else { continue }
             infos.append(info)
         }
     }
     
-    /// Creates an offline map info for a given URL.
-    private func makeOfflineMapInfo(url: URL) -> OfflineMapInfo? {
-        let infoURL = url.appending(components: "info.json")
-        guard FileManager.default.fileExists(atPath: infoURL.path()) else { return nil }
-        Logger.offlineManager.debug("Found offline map info at \(infoURL.path())")
-        guard let data = try? Data(contentsOf: infoURL) else { return nil }
-        return try? JSONDecoder().decode(OfflineMapInfo.self, from: data)
-    }
-    
     /// Saves the map info to the pending folder for a particular portal item.
     /// The info will stay in that folder until the job completes.
     private func savePendingMapInfo(for portalItem: PortalItem) async {
         guard let portalItemID = portalItem.id,
-              !offlineMapInfos.contains(where: { $0.portalItemID == portalItemID }),
-              let info = OfflineMapInfo(portalItem: portalItem)
+              !offlineMapInfos.contains(where: { $0.portalItemID == portalItemID })
         else { return }
         
         // First create directory for what we need to save to json
         let url = URL.pendingMapInfoDirectory(forPortalItem: portalItemID)
-        let infoURL = url.appending(path: "info.json")
-        
-        Logger.offlineManager.debug("Saving pending offline map info to \(url.path())")
         
         // If already exists, return.
-        let completedDir = URL.portalItemDirectory(forPortalItemID: portalItemID).appending(path: "info.json")
-        guard !FileManager.default.fileExists(atPath: completedDir.path()) else {
-            Logger.offlineManager.debug("Returning, info already exists in job completion directory: \(completedDir.path())")
-            return
-        }
-        guard !FileManager.default.fileExists(atPath: infoURL.path()) else {
-            Logger.offlineManager.debug("Returning, info already exists in pending directory: \(url.path())")
+        guard OfflineMapInfo.doesInfoExists(at: url) else {
```

Comment:
```suggestion
        guard !OfflineMapInfo.doesInfoExists(at: url) else {
```
This change seems to work so that the info is saved for the first map area. For subsequently saved map areas for a web map, `doesInfoExists()` is not called due to the other guard statements in `savePendingMapInfo()`. Is this check `guard !OfflineMapInfo.doesInfoExists(at: url)` necessary?